Python/bytecodes.c

```diff
@@ -3743,11 +3743,10 @@ dummy_func(
             return frame;
         }
 
-        op(INSERT, (stuff[oparg], top -- top, stuff[oparg])) {
-            // Inserts TOS at position specified by oparg
-            PyObject *tos = TOP();
+        op(INSERT, (unused[oparg], top -- top, unused[oparg])) {
+            // Inserts TOS at position specified by oparg;
             for (int i = 1; i < oparg + 1; i++) {
-                stack_pointer[i] = stack_pointer[i - 1];
+                stack_pointer[-i] = stack_pointer[-(i - 1)];
```

Comment:
I still think this is incorrect.
- At i = 1, we do sp[-1] = sp[0], which reaches beyond the stack
- At i = 2, we do sp[-2] = sp[-1], which copies whatever we got from sp[0]
- At i = 3, we do sp[-3] = sp[-2], which copies, etc.

Reply:
Thanks for reasoning this out. I decided to use `memmove` instead.